src/services/apiService.js

```diff
@@ -188,6 +187,14 @@ angular.module('openshiftCommonServices')
     return resource;
   }
 
+  function kindToResourceGroupVersion(kind) {
+    var resourceStr = kindToResource(kind.kind);
```

Comment:
I'm confused by this. I'd expcct this to take in a string like `"ReplicaSet"` rather than an object. We might think about the function and arg name here.